### PR TITLE
Post-Getting-Started comments/edits

### PR DESCRIPTION
**Pull request by filmaj**
This is just a starting point for a discussion. Needs some work before merging!

# Summary

1. The index / landing page reads like the docs site just went up. Don't think it stands the test of time. Edited that to try to make it more generic and open-ended.
2. Expanded on README's directory structure section. Definitely need help from y'all to properly describe the directories that were missed!

# Further Comments

Thinking about how to unify the PhoneGap developer experience, perhaps it would be useful to outline the vision for the development workflow right at the start. One can implicitly understand, by looking at the steps (install desktop app or cli, then install the mobile app, and thus, use one to test your app on the other). Perhaps outlining the dev ‘flow’ vision here, but right at the start, would be helpful? Further to this point, this opening sentence from step 4 of the getting started guide is, more or less, exactly that: “You can use the PhoneGap Developer App paired with PhoneGap Desktop to immediately preview your app on a device without installing platform SDKs, registering devices, or compiling code.”

### CODE REVIEW

File: docs/0-index.html.md [hunk 1]
```diff
@@ -6,17 +6,14 @@ next: 1-getting-started/1-install-phonegap/1-desktop.html.md
 ---
 <img style="float:right; width: 50%; max-width: 250px;" src="/images/excited_robit.svg" alt="Excited Robot">
 <p style="color:#888888;font-size:1.35rem;line-height:1.9rem;">
-Welcome to the new version of the PhoneGap Docs! We're working on putting together
-a comprehensive resource that covers all things PhoneGap and mobile app development,
-that is both in-depth and easy to follow.
+These docs aim to be a comprehensive resource that covers all things PhoneGap
+and mobile app development, that is both in-depth and easy to follow.
```
Comment (hollyschinsky) [inline docs/0-index.html.md]: Another idea... 
>These docs aim to be a comprehensive resource covering PhoneGap and associated tooling, as well as some general mobile app development concepts we think developers will find helpful.

File: docs/0-index.html.md [hunk 2]
```diff
@@ -6,17 +6,14 @@ next: 1-getting-started/1-install-phonegap/1-desktop.html.md
...
-We didn't want to keep you waiting for all the good stuff, so we've published
-some key resources to get you going. If you are new to PhoneGap, we recommend
-beginning by working through our <a href="/getting-started/1-install-phonegap/desktop">Getting Started Guide</a>.
-We've also put together a helpful <a href="/tutorials/develop/hello-world-explained/">tutorial</a> on extending our
-"Hello World" example app. These should get you going, and keep your eyes peeled
-for new material to come soon.
+If you are new to PhoneGap, we recommend beginning by working through our
+<a href="/getting-started/1-install-phonegap/desktop">Getting Started Guide</a>.
+We've also put together a helpful <a href="/tutorials/develop/hello-world-explained/">tutorial</a>
+on extending our "Hello World" example app.
```
Comment (hollyschinsky) [inline docs/0-index.html.md]: Sounds good

File: docs/0-index.html.md (hunk 1 above)
Comment (filmaj) [inline docs/0-index.html.md]: Love it!

**Comment by GarthDB** [comment]
So is this good now @hollyschinsky or do you want to review it further?

**Pull request merged**